tests/web_api/test_calculate.py

```diff
@@ -1,28 +1,23 @@
-# -*- coding: utf-8 -*-
-
-import os
+import copy
+import dpath
 import json
-from http.client import OK, BAD_REQUEST, NOT_FOUND, INTERNAL_SERVER_ERROR
-from copy import deepcopy
-
+from http import client
+import os
 import pytest
-import dpath
 
 from openfisca_country_template.situation_examples import couple
 
-from . import subject
-
 
-def post_json(data = None, file = None):
+def post_json(_client, data = None, file = None):
```

Comment:
Why do we prefix the variable with an underscore, to avoid a collision with the fixture?

Reply:
Ah no, because it is called `test_client`, why do you think then the prefix to be needed?

Reply:
Sorry I overlooked the import of `client` from the `http` package 😄  do they collide in the scope of this function?

Reply:
Mm I was indeed trying to avoid collision with `http.client`. But upon testing I realise that they don't actually collide within the scope of this function.

Do you think (just to avoid confusion) I should choose a totally different name for the `_client` argument? Or do you think naming it `client` (without underscore) is fine?

Reply:
@RamParameswaran Hello! I would personally™ just avoid the underscore, as when I'm looking at the code from a naive perspective, I assume the argument is called `_something` just for informative reasons —instead of say just `_`— but that it is not meant to be used. Then, when I look at the content of the function, I'm surprised that the argument is actually being used. Whether it is called _client_, _test_client_, or something else is actually indifferent to me 😃 and I'm OK with whatever you think is better suited for reader's comprehension.

Reply:
OK. I've removed the underscore.